### PR TITLE
[rllib] fix clip by value issue as TF upgraded

### PR DESCRIPTION
**Pull request by joneswong**
## What do these changes do?
enable `tf_run_builder` output meaningful error message
fix `clip_by_value` error as TF upgraded

## Related issue number
#4691

## Linter

- [x] I've run `scripts/format.sh` to lint the changes in this PR.

### CODE REVIEW

File: python/ray/rllib/utils/tf_run_builder.py [hunk 1]
```diff
@@ -47,7 +47,8 @@ def get(self, to_fetch):
                 self._executed = run_timeline(
                     self.session, self.fetches, self.debug_name,
                     self.feed_dict, os.environ.get("TF_TIMELINE_DIR"))
-            except Exception:
+            except Exception as ex:
+                logger.info(ex)
```
Comment (ericl) [inline python/ray/rllib/utils/tf_run_builder.py]: logger.exception()?

Also, I thought that in Python the cause of an exception is shown along with a newly raised exception. So you should see "ValueError"... with "Exception" above somewhere in the trace? Is that not what you're seeing?

**Review by ericl — APPROVED** [review]

File: python/ray/rllib/utils/tf_run_builder.py (hunk 1 above)
Comment (joneswong) [reply python/ray/rllib/utils/tf_run_builder.py]: I resolved the conflicts and modified according to your suggestions. Without sth like `logger.exception(...)`, in our case (described in the related issue), there is no information that is helpful for exactly locating the causes.

**Comment by joneswong** [comment]
@ericl  Hi eric, how is this pr going? I fetched the latest upstream/master and found that there is still shape issue of the `clip_by_value` function.

**Comment by ericl** [comment]
jenkins retest this please

**Comment by ericl** [comment]
Merging, thanks!

**Pull request merged**